Assets/AdminMap/Scripts/GamePanel/Map.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections;
```

Comment:
Could you overwrite the Library/Map.cs file with this or does it have to be placed here? It would be nice to have it directly in Assets as it probably will be utilized by a lot of components.